plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -123,13 +134,17 @@ export const Modal = () => {
   return (
     <>
       <DialogTitle>
-        <Paper className={classes.container}>
+        <Box className={classes.dialogTitle}>
           <SearchBar
             className={classes.input}
             inputProps={{ ref: searchBarRef }}
             onKeyDown={handleSearchBarKeyDown}
           />
-        </Paper>
+
+          <IconButton aria-label="close" onClick={toggleModal}>
```

Comment:
It shouldn't be optional, a dialog without a close button is not user-friendly.

Reply:
Previously people could close the modal by clicking on its backdrop or use the `ESC` key to close it via keyboard navigation, so they could prefer not to have the close button. It might be a good idea to have a close button optionally.

Reply:
I'm not an expert in accessibility but I'm pretty certain dialogs without a close button is a bad practice, we can't expect every user to know that the backdrop click closes the dialog, in fact, I've met a few sites where backdrop click did nothing (whether it's a bug or intended idk)...

Reply:
I agree that it's good to be clear on how to close the modal. We can open an issue later if one of us really needs to make the button optional, but let's keep it until then 🙂